modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/edit_form/start.jsp

```diff
@@ -18,7 +18,7 @@ String fullName = namespace.concat(HtmlUtil.escapeAttribute(name));
 
 		<div aria-orientation="vertical" class="panel-group panel-group-flush" role="tablist">
 			<c:if test="<%= Validator.isNotNull(onSubmit) %>">
-				<div aria-label="<%= HtmlUtil.escape(portletDisplay.getTitle()) %>" class="input-container" role="group">
+				<div aria-labelledby="<%= HtmlUtil.escape(portletDisplay.getTitle()) %>" class="input-container" id="<%= namespace %>groupLabel" role="group">
```

Comment:
This does not look right. Looking at [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby), it should reference `id` of an element. Here, we are setting it to title string. The same logic is for `general` string.

Reply:
@markocikos, thanks for bringing this up, I can see what you mean.

@veroglez, as you originally provided the fix/recommendation for this, can you verify if this is what you meant as a fix?
If not, can you provide further info on how we can make the correct changes.


This video shows the JAWS behavior with the changes.
**Down arrowing through menu**

https://github.com/liferay-frontend/liferay-portal/assets/31672551/bdd2211e-984a-46f6-9450-a4e605bb56b7


**Tabbing through menu**

https://github.com/liferay-frontend/liferay-portal/assets/31672551/5f840eda-658c-4c97-a45e-acdd65a8977e

Reply:
Hello @markocikos,
After taking another look, it seems like I can leave the label as is and instead change the title to the name of the Section. I will do this by making the change in the JSP.